core/src/main/java/io/grpc/ServerCall.java

```diff
@@ -57,21 +53,6 @@
 @DoNotMock("Use InProcessTransport and make a fake server instead")
 public abstract class ServerCall<ReqT, RespT> {
   /**
-   * {@link Attributes.Key} for the remote address of server call attributes
-   * {@link ServerCall#attributes()}
-   */
-  @ExperimentalApi("https://github.com/grpc/grpc-java/issues/1710")
-  public static final Attributes.Key<SocketAddress> REMOTE_ADDR_KEY =
```

Comment:
Leave these in-place (just copying from `CanonicalAttributes`) with `@Deprecated` for one release cycle?